qiskit/pulse/schedule.py

```diff
@@ -132,7 +140,7 @@ def instructions(self):
         def key(time_inst_pair):
             inst = time_inst_pair[1]
             return (time_inst_pair[0], inst.duration,
-                    min(chan.index for chan in inst.channels))
+                    sorted(chan.name for chan in inst.channels))
```

Comment:
What is this for? Was there a bug?

Reply:
There was a bug if two channels were of the same index but different indices the order was not guaranteed.